### PR TITLE
[O11y][Hadoop] Migrate `node_manager` and `data_node` dashboards visualizations to lens in kibana 8.3.0

### PR DESCRIPTION
**Pull request by harnish-elastic**
- Enhancement

## What does this PR do?

- Migrate visualizations to lens for Hadoop integration package

- Statistics for **Hadoop** Lens migration:

<html>
<body>
<google-sheets-html-origin>

Hadoop dashboards | Before Migration |   | After Migration |  
-- | -- | -- | -- | --
  | Lens | Visualization | Lens | Visualization
[Metrics Hadoop] Data nodes | 6 | 3 | 9 | 0
[Metrics Hadoop] Node Manager | 0 | 3 | 3 | 0
  | 6 | 6 | 12 | 0

</body>
</html>

## Checklist

- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that panels are populated with data.
- [x] I have verified that panels are not distorted after being migrated to lens.
- [x] I have updated screenshots of dashboard.
- [x] I have verified that data count are matching and panel aggregations are same as before.

## Author's checklist

- [x] Migrated panels should be removed from visualization folder.
- [x] Migrated visualizations are populating in current Kibana version 8.3.0. 

## Related issues

- Relates #5221

### CODE REVIEW

**Review by kush-elastic — APPROVED** [review]
LGTM!

File: packages/hadoop/changelog.yml [hunk 1]
```diff
@@ -1,4 +1,9 @@
 # newer versions go on top
+- version: "0.5.0"
+  changes:
+    - description: Migrate `[Metrics Hadoop] Data nodes` and `[Metrics Hadoop] Node Manager` dashboard visualizations to lens.
```
[the remark this answers is not in the document]
Comment (harnish-elastic) [reply packages/hadoop/changelog.yml]: Updated, thanks!

**Comment by agithomas** [comment]
image

I assume, the panel at the top is the time duration it takes .
In the panel below, does the "Duration" text in the "Container Launch Duration (Num of operation) " make sense? Please change from `Num of operation` to `Operation Count`.

**Comment by harnish-elastic** [comment]
> image
> 
> I assume, the panel at the top is the time duration it takes . In the panel below, does the "Duration" text in the "Container Launch Duration (Num of operation) " make sense? Please change from `Num of operation` to `Operation Count`.

This name is already there in the current codebase. This is just the lens migration PR so we won't be changing any behavior as a part of this PR otherwise it will mess it up. However, If you strongly feel that this change needs to be done then we can raise a separate issue for that and track them separately.

**Comment by agithomas** [comment]
> This name is already there in the current codebase. This is just the lens migration PR so we won't be changing any behavior as a part of this PR otherwise it will mess it up. However, If you strongly feel that this change needs to be done then we can raise a separate issue for that and track them separately.

As you think that the change will complicate the PR, continue with the same scope.

**Review by agithomas — APPROVED** [review]
LGTM

**Pull request merged**